apps/user-portal/src/components/multi-factor-authentication/multi-factor-authentication.tsx

```diff
@@ -36,17 +36,17 @@ export const MultiFactorAuthentication: React.FunctionComponent<MfaProps> = (pro
 
     return (
         <SettingsSection
-            description={ t("views:sections.mfa.description") }
-            header={ t("views:sections.mfa.heading") }
+            description={t("views:sections.mfa.description")}
+            header={t("views:sections.mfa.heading")}
         >
             <List divided verticalAlign="middle" className="main-content-inner">
                 <List.Item className="inner-list-item">
-                    <SMSOTPAuthenticator onNotificationFired={ onNotificationFired }/>
+                    <SMSOTPAuthenticator onNotificationFired={onNotificationFired} />
```

Comment:
I think we agreed on adding a space between the curly brackets and the characters. So shall we make it consistent?